src/main/java/com/tradeshift/service/MessageService.java

```
@@ -0,0 +1,25 @@
+package com.tradeshift.service;
+
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Component;
+
+import com.tradeshift.dto.Message;
+import com.tradeshift.exception.ServiceException;
+
+@Component
+public class MessageService {
+
+    private static final Logger logger = Logger.getLogger(MessageService.class);
+
+    public Message createMessage(String name) throws ServiceException {
+
+        try {
+            Message message = new Message(name);
+            message.createMessageContent();
+            return message;
+        } catch (Exception ex) {
```

is it on purpose that you catch all Exceptions?